src/main/java/Duke.java

```diff
@@ -1,7 +1,9 @@
 import java.util.Scanner;
 
 public class Duke {
-    private static Task[] list = new Task[100];
+    public static final String INITIALISING = "initialising";
```

Comment:
Your code looks good and is pretty easy to read. I like how you have used different methods and not put all of your code in the main method.